test/unit/LedgerId.js

```diff
@@ -0,0 +1,117 @@
+import * as hex from "../src/encoding/hex.js";
+import LedgerId from "../src/LedgerId.js";
+
+const MAINNET = "mainnet";
+const HEX_MAINNET = hex.encode(new Uint8Array([0]));
```

Comment:
Not blocking this PR for this, but I'm curious why you use `hex.encode(...)` here instead of just constructing the hex version yourself `"00"`

Reply:
🤷